_config.yml

```diff
@@ -641,3 +641,18 @@ jsonresume:
   - languages
   - interests
   - references
+
+# Customize the section titles in the cv page, leave it blank for default
+sectiontitles:
```

Comment:
I believe it's best to use `sections_titles` instead. `jsonresume` is only in a single word because it is the name of the library.